tmt/schemas/results.yaml

```diff
@@ -59,9 +59,35 @@ items:
           type: string
 
     log:
+      $ref: "/schemas/common#/definitions/result_log"
+
+    checks:
       type: array
       items:
-        type: string
+        type: object
+        properties:
+          name:
+            type: string
+
+          event:
+            type: string
+            enum:
+              - "before-test"
```

Comment:
this  needs update, we also have `before-all-tests` and `after-all-tests`

Reply:
No, we don't :) We **will** have them, but we don't, yet.